### DISCUSSION TITLE
GitHub Swags

### DISCUSSION THREAD

**Discussion by git-sumana** [community/community#110984] [New to GitHub] [answered]
Are there any new updates for getting free swags of GitHub community or any merchandise??

**Comment by SebastienXia** [comment]
I don't think so, but there exist [THE GITHUB SHOP](https://www.thegithubshop.com/), where there are a lot goodies and marchandises !

![téléchargement](https://github.com/community/community/assets/105004912/57d512a5-fce9-476e-9bde-b8c01521d7e5)

**Reply by rolbxl** [reply, marked as answer]
That is Swag! 🦄

**Reply by FAHADHARTY** [reply]
maram.b snapped

**Reply by FAHADHARTY** [reply]
I'm ``~~~~
```
```

**Reply by FAHADHARTY** [reply]
~~~~ maram.bj

**Comment by Sameer8976fkhd** [comment]
![Uploading 1000012488-01.jpg…]()
